pantheon/utils/utils.py

```diff
@@ -14,6 +14,18 @@ def getLimits(headers):
         return (appLimits,methodLimits)
     return None
 
+def serverToRegion(server):
+    americas = ["br1", "la1", "la2", "na1", "oc1"]
```

Comment:
oc1 should return 'sea', not americas
```suggestion
    sea =  ["ph2", "sg2", "th2", "tw2", "vn2", "oc1"]
    americas = ["br1", "la1", "la2", "na1"]
```